torch/tensor.py

```diff
@@ -426,6 +426,12 @@ def __array_wrap__(self, array):
             array = array.astype('uint8')
         return torch.from_numpy(array)
 
+    def __contains__(self, val):
```

Comment:
You can add the docstring here, instead of having to add it in a separate file.

Reply:
Thanks.